api/test/trace/noop_test.cc

```diff
@@ -31,6 +32,13 @@ TEST(NoopTest, UseNoopTracers)
   s1->SetAttribute("abc", 4);
 
   s1->AddEvent("abc");  // add Empty
+  SpanContext sp(false, false);
+  s1->AddLink(sp);
+  using M = std::map<std::string, std::string>;
+  M m1    = {{"abc", "123"}, {"xyz", "456"}};
+  opentelemetry::trace::KeyValueIterableView<M> iterable{m1};
+
+  s1->AddLink(sp, iterable);
```

Comment:
Having the additional `AddLink` methods mentioned above will make this easier, and you could do `s1->AddLink(sp, {{"abc", "123"}, {"xyz", "456"}});`.